pages/dev/c/_communityname/index.vue

```diff
@@ -73,7 +73,7 @@
       :locationToDisplay="`#${this.communityName}`"
```

Comment:
This was just missing it's hidden property for desktop, minor thing from Mobile Refactor